app/forms/state_file/nc_state_id_concern.rb

```diff
@@ -0,0 +1,36 @@
+module StateFile
+  module NcStateIdConcern
```

Comment:
I'm aware that there's a `StateIdForm`. I chose to do things this way for a few reasons. `StateIdForm` seems to have a lot of crossed logic from multiple states and I wasn't sure if I wanted or needed to contribute to that. I also didn't want to create a form which _could_ be used to map to a model but shouldn't (a concern is unusable by itself) and lastly I needed to break out into my own code to take advantage of `DateAccessor` for simpler validation of dates.